src/popup/pages/Names/AuctionList.vue

```diff
@@ -154,7 +154,6 @@ export default defineComponent({
 
 .auction-list {
   .toolbar {
-    --background: var(--screen-bg-color);
```

Comment:
![image](https://github.com/user-attachments/assets/1fb467c7-e8a1-42a0-91e2-3eb331f8b41b)

Reply:
OK, I brought back the variable.